tests/TestsBase.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-use Orchestra\Testbench\TestCase;
 use Dimsav\Translatable\Test\Model\Country;
+use Orchestra\Testbench\TestCase;
 
 class TestsBase extends TestCase
 {
     protected $queriesCount;
 
-    const DB_NAME     = 'translatable_test';
+    const DB_NAME = 'translatable_test';
```

Comment:
please revert this

Reply:
what's the strange formatting  ? 😄 seems changes are ok